test/bin/mdx-test/failure/in-toplevel/test-case.md.expected

```diff
@@ -1,2 +1,2 @@
-Error in the OCaml file include code block in test-case.md at line 4:
+File "test-case.md", lines 3-4: Error in the OCaml file include code block
```

Comment:
I had a quick look at the test file here which looks like this:
````markdown
Testing error message

```ocaml file=not_found.ml
# 1 + 1;
```
````

It seems like the closing line isn't counted as part of the block, location-wise. Do you know if this is somehow expected? Do you we could fix it so that the loc also includes that last line?

Reply:
I fixed this, but it also push back the locations in the directive for mdx rule, not sure if this is very important.